tests/monotouch-test/ARKit/ARConfigurationTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿#if XAMCORE_2_0 && __IOS__
+
+using System;
+using System.Reflection;
+using ARKit;
+using Foundation;
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.ARKit {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class ARCondigurationTest {
+
+		[SetUp]
+		public void Setup ()
+		{
+			TestRuntime.AssertXcodeVersion (10,0);
```

Comment:
```suggestion
			TestRuntime.AssertXcodeVersion (9,3);
```

Given that those APIs are `[iOS (11,3)]`